shadow-dom/declarative/support/helpers.js

```diff
@@ -0,0 +1,4 @@
+function setInnerHTML(el,content) {
+  const fragment = (new DOMParser()).parseFromString(`<pre>${content}</pre>`, 'text/html', {includeShadowRoots: true});
```

Comment:
This is a non-standard API so I don't think this works.

Reply:
Right - this is the old version of setHTMLUnsafe(), and it won’t be standardized.

Reply:
Only chrome added the includeShadowRoots parameter, right?
So these tests would have never worked before my test change anyway?

Reply:
WebKit had a preffed-off implementation, but I removed it.

I think @gsnedders is okay with closing this PR so I'll do that to avoid further confusion.